argus.js

```diff
@@ -12,8 +12,13 @@ const ArgusJS = function(token) {
     }, false);
 
     return {
-        token: () => currentToken,
-        get: (url) => fetch(url, { "method": "GET", "headers": { "Authorization": "Token " + currentToken } })
+        get: (url) => fetch(`${mdr_url}${url}`, { "method": "GET", "headers": { "Authorization": "Bearer " + currentToken.access } }),
+        getById: (url, id) => fetch(`${mdr_url}${url}/${id}`, { "method": "GET", "headers": { "Authorization": "Bearer " + currentToken.access } }),
```

Comment:
I don't think we need this function, and it might be a little confusing to users to have to specify the URL and id separately. Same for `put`, `patch` and `delete` - just take a URL (and data if relevant).